mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -500,8 +500,27 @@
     <!-- New Onboarding flow first screen subtitle the one that appears under title -->
     <string name="onboarding_first_screen_subtitle">Fast. Private. No distractions.</string>
 
+    <!-- New Onboarding flow first screen 'terms of use' text -->
+    <!--  %1$s will be replaced by onboarding_first_screen_terms_of_use_link -->
+    <string name="onboarding_first_screen_terms_of_use_text" tools:ignore="UnusedResources">By continuing, you agree to the %1$s</string>
+
+    <!-- New Onboarding flow first screen 'terms of use link' -->
+    <!-- 'Firefox' should NOT be translated -->
```

Comment:
In this specific case, some locales may have to adapt the noun given its place in the sentence - so we can delete the comment